### DISCUSSION TITLE
faq/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#327] [docs-comment]
# faq/

本文主要包含了使用主题的过程中可能会遇到的常见问题与解决方法。 如果您遇到任何问题，您可以先在以下内容中寻找是否有相关的问题和解决方法。 如果没有找到，您可以先通过 Github Discussions 与我们讨论。 如果您确定某处确实存在问题，请在 Github Open an issue, 在 Issue 中说明该问题的具体细节，如有必要，请尽量提...

https://theme-plume.vuejs.press/faq/

**Comment by paiad** [comment]
1

**Comment by paiad** [comment]
作者我有个问题，为什么我想加入一个鼠标点击的插件在config.ts中就是加不上去？

**Reply by pengzhanbo** [reply]
麻烦表述清楚，必要时贴上相关的代码。

**Reply by paiad** [reply]
就是在config.ts中插入这个背景音乐的插件
```
export default defineUserConfig({
    plugins: [
        [
            '@vuepress-reco/vuepress-plugin-bgm-player',
            {
                audios: [
                    {
                        name: 'LOSER',
                        artist: '米津玄師',
                        url: 'https://www.ytmp3.cn/down/73654.mp3',
                        cover: 'https://p1.music.126.net/qTSIZ27qiFvRoKj-P30BiA==/109951165895951287.jpg?param=200y200'
                    }
                ] ,
                // 是否默认缩小
                autoShrink: true ,
                // 缩小时缩为哪种模式
                shrinkMode: 'float',
                // 悬浮窗样式
                floatStyle:{ bottom: '10px', 'z-index': '999999' }
            }
        ]
    ],
```报错。我不知道以何种方式添加这个插件。

**Reply by pengzhanbo** [reply]
这种方式是 `vuepress@1.x` 的配置方式了， `vuepress@2.x` 已经不支持这么写的，查看 [新的文档](https://v2.vuepress.vuejs.org/zh/guide/plugin.html)

**Comment by paiad** [comment]
作者有个问题，笔记页的导航面包屑在哪里进行配置啊？

**Reply by paiad** [reply]
不用了up，我已经自己解决了。🐶

**Comment by paiad** [comment]
作者我有个问题，这个笔记右侧展示的只能到h3么？h4之后的都不展示么？

**Reply by pengzhanbo** [reply]
用 [outline](https://theme-plume.vuejs.press/config/basic/#outline) 可以控制。

**Reply by paiad** [reply]
作者，如果我想在此项目中做一个可以运行python的代码段可以怎么入手呢？

**Reply by pengzhanbo** [reply]
通过 fork 本仓库后，往 `plugins/plugin-md-power` 目录中添加相关代码，然后通过 PR 提交回本仓库。在此之前可以先开一个 feature request 的 issue 进行具体的方案讨论，也可以在 PR 里直接描述。在 PR 中，也需要在`docs` 中添加该功能的说明文档。仓库根目录下的 `contributing.md` 有说明如果开始参与项目贡献。

**Reply by paiad** [reply]
作者对于文本的各种高亮显示除了反应号``和<mark></mark>的样式还有其他的高亮样式么？

**Reply by pengzhanbo** [reply]
有很多， 比如 `==xxx==`, `!!xx!!`, `**xx**`，相关 markdown 语法参考 [markdown](https://theme-plume.vuejs.press/guide/markdown/basic/) 以及同级的说明文档。

**Comment by kingmingjie** [comment]
作者大大，如果想要做文章后台管理，有推荐的方案吗
